src/constants/translations/ua/index.js

```diff
@@ -8,9 +8,24 @@ import login from './login.json'
 import modals from './email-modals.json'
 import breadCrumbs from './breadcrumbs.json'
 import completeProfile from './complete-profile.json'
+import findOffers from './find-offer-page.json'
+import findOffersTooltips from './find-offer-page-tooltip.json'
 
 const ua = {
-  translations: { common, errorPage, button, header, questions, titles, login, modals, breadCrumbs, completeProfile }
+  translations: {
+    common,
+    errorPage,
+    button,
+    header,
+    questions,
+    titles,
+    login,
+    modals,
+    breadCrumbs,
+    completeProfile,
+    findOffers,
+    findOffersTooltips
```

Comment:
move this tooltips translations to findOffers file, you dont need separate file for this

Reply:
fixed